code/game/objects/effects/cult/rituals/tear_spawners/pillar_spawner.dm

```diff
@@ -0,0 +1,67 @@
+
+/obj/effect/cult_ritual/tear_spawners/pillar_spawner
+	finished = TRUE
+	var/obj/structure/cult/pillar
+
+/obj/effect/cult_ritual/tear_spawners/pillar_spawner/New(turf/loc, datum/rune_spell/tearreality/_source, _direction)
+	if (!_direction)
+		qdel(src)
+		return
+	..()
+	if (source)
+		source.pillar_spawners += src
+	direction = _direction
+
+	for(var/direc in GLOB.cardinals)
+		var/turf/T = get_step(src, direc)
+		near_turfs += T
+		var/turf/U = get_step(T, direc)
+		far_turfs += U
+
+	for(var/direc in GLOB.diagonals)
+		var/turf/T = get_step(src, direc)
+		far_turfs += T
+
+/obj/effect/cult_ritual/tear_spawners/pillar_spawner/move_step()
+	return
+
+/obj/effect/cult_ritual/tear_spawners/pillar_spawner/execute(level)
+	switch(level)
+		if (1)
+			var/turf/T = loc
+			if (T)
+				T.narsie_act()
```

Comment:
this causes harvesters to spawn if any players are unlucky enough to be on the turf